### ISSUE TITLE
firejail sylpheed doesn´t work properly

### ISSUE THREAD

**Issue by Rosika2** [netblue30/firejail#3122]
Hi altogether,

after a firejail-update to version 0.9.62 I encountered a new problem. This time with **sylpheed**.

`firejail --private=/media/rosika/f14a27c2-0b49-4607-94ea-2e56bbf76fe1/DATEN-PARTITION/Dokumente/kgw_work3_sylpheed/ sylpheed`

used to work but not anymore.
I get the following message:

```
Reading profile /etc/firejail/sylpheed.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-interpreters.inc
Reading profile /etc/firejail/disable-passwdmgr.inc
Reading profile /etc/firejail/disable-programs.inc
Warning: networking feature is disabled in Firejail configuration file
Parent pid 13980, child pid 13981
Warning: cleaning all supplementary groups
Warning: cleaning all supplementary groups
Child process initialized in 408.11 ms

(sylpheed:5): GLib-GIO-CRITICAL **: 17:00:41.923: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed

(sylpheed:5): GLib-GIO-CRITICAL **: 17:00:41.923: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed

(sylpheed:5): GLib-GIO-CRITICAL **: 17:00:41.923: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed

(sylpheed:5): GLib-GIO-CRITICAL **: 17:00:41.923: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed
/home/rosika/Mail/inbox: chdir: Keine Berechtigung  # not permitted

(sylpheed:5): LibSylph-WARNING **: 17:00:44.049: [17:00:44] Kann Datei nicht schreiben.  # cannot write file

/home/rosika/Mail/queue: chdir: Keine Berechtigung  # not permitted


```
The GUI opens up but the programme gets stuck when trying to download new e-mails. A popup window says _"cannot write file"_.

What **does** work however is "firejail --noprofile [...]".
So is has got something to do with **sylpheed.profile**.

Greetings
Rosika

**Comment by rusty-snake** [comment]
Last commit on sylpheed.profile:  92bff8a on 17 Oct 2018.
However it is 109b1865 (noblacklist no blacklist).
Now `~/Mail` is blacklisted (and a few more https://github.com/netblue30/firejail/commit/109b1865669ac1038454023762fb83cd0ebf6cca#diff-2628c1c1a167c5e64c49db36d3154ccdR417)

TL;DR: add to sylpheed.profile
```
noblacklist ${HOME}/Mail
```

Is `~/Mail` your choice or program default?

**Comment by Rosika2** [comment]
@rusty-snake:
Hello again and thanks a lot for your quick help.

The addition of  `noblacklist ${HOME}/Mail` did the trick. Sylpheed works fine again.

> Is ~/Mail your choice or program default?

I didn´t change anything when setting up sylpheed. So I´d have to say: **programm default**.

Thank you very much again. Your help is much appreciated.

Greetings.
Rosika

**Comment by glitsj16** [comment]
@Rosika2 After deliberation it would be more reasonable to re-write sylpheed.profile as a redirect profile. I've done so in https://github.com/netblue30/firejail/commit/3e66d6c8ed229552546bc4d9c93a805f40825bbf just now. Would you be so kind to test your sylpheed installation still works as expected with these changes please?

**Comment by rusty-snake** [comment]
@Rosika2 simply copy the sylpheed profile from git master, there are no big chages since 0.9.62.

@glitsj16 more semantically idiomatic would be `nowhitelist /usr/share/doc/claws-mail`

**Comment by glitsj16** [comment]
@rusty-snake Nice one, I'll edit the profile accordingly :+1:.

**Comment by Rosika2** [comment]
@rusty-snake:
@glitsj16:

O.K., I did it this way:

> simply copy the sylpheed profile from git master [...]

I can confirm that it works just fine. The new sylpheed.profile as a redirect profile seems to be a fine solution as well.
Thanks for all the work of yours.

Greetings.
Rosika

**Comment by glitsj16** [comment]
@Rosika2 Great. Thank you for confirming, always appreciated!
